src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandStatus.java

```diff
@@ -47,5 +47,18 @@ public enum CommandStatus {
    * is 'true' and no document available in DB for matching condition
    */
   @JsonProperty("upsertedId")
-  UPSERTED_ID;
+  UPSERTED_ID,
+  /**
+   * The element has the session id of offline writer, which is used to write the data offline to
+   * the SSTable files for example
+   */
+  @JsonProperty("offlineWriterSessionId")
+  OFFLINE_WRITER_SESSION_ID,
```

Comment:
Not a big deal but this file has been maintained in alphabetic order.

Reply:
Changes done in this commit [fb57ad3](https://github.com/stargate/data-api/pull/905/commits/fb57ad39c79f6eafb7878e785744a8bb5a2bfe07).